examples/granola/honcho_granola.py

```
@@ -0,0 +1,1096 @@
+            transcript = await granola.get_meeting_transcript(mid)
+            if transcript:
+                m["transcript"] = transcript
+                fetched_transcript = True
+
+            try:
+                details = await granola.get_meeting_details(mid)
+                m.update(details)
+                if extract_summary_from_meeting(m):
+                    fetched_summary = True
+            except Exception:
+                pass
```

Ruff (0.15.5)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)